src/screens/profile/profile.tsx

```diff
@@ -1,19 +1,28 @@
-import React, { useState } from 'react';
+import React, { useCallback, useRef, useState } from 'react';
 import { View } from 'react-native';
 import { useNavigation } from '@react-navigation/native';
-import { FAB } from 'react-native-paper';
 import { SafeAreaView } from 'react-native-safe-area-context';
+import { Divider, FAB } from 'react-native-paper';
+import PagerView from 'react-native-pager-view';
 
+import { Navbar } from 'src/screens/profile/components/navbar';
 import { ProfileRoute } from 'src/common/enums';
 import { Text } from 'src/components';
 import addActions from './add-actions';
 import styles from './styles';
 
-type IProfileScreenProps = Record<string, never>;
+type ProfileScreenProps = Record<string, never>;
 
-const ProfileScreen: React.FC<IProfileScreenProps> = () => {
+const ProfileScreen: React.FC<ProfileScreenProps> = () => {
   const navigation = useNavigation();
   const [addMenuOpen, setAddMenuOpen] = useState<boolean>(false);
```

Comment:
Let's avoid unnecessary types
```suggestion
  const [addMenuOpen, setAddMenuOpen] = useState(false);
```